content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation](/tracing/trace_collection/) is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation](/tracing/trace_collection/automatic_instrumentation/) is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries](https://opentelemetry.io/docs/concepts/instrumentation/libraries/) from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
+
+## Prerequisites
+
+Before adding OpenTelemetry instrumentation libraries set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
+
+<div class="alert alert-warning">
+When replacing an existing Datadog instrumentation with its OpenTelemetry equivalent, remember to disable the
+Datadog instrumentation to avoid duplicate spans in the trace.
+</div>
+
+## Language support
+
+| Language | Minimum version required |
+|----------|--------------------------|
+| Java     | v1.35.0                  |
+| Python   | v2.10.0                  |
+| Ruby     | v2.1.0                   |
+| Go       | TBA                      |
+| Node.js  | TBA                      |
+| PHP      | TBA                      |
+| .NET     | TBA                      |
+
```

Comment:
```suggestion
| Go       | Not supported                      |
| Node.js  | Not supported                      |
| PHP      | Not supported                      |
| .NET     | Not supported                      |

```
Small tweak, but I'd suggest using "Not supported", "N/A", or " - " instead of TBA. This ensures we're documenting the current state of support rather than implying future functionality.

Reply:
On reflection `Not supported` is not quite right because the functionality is actually released, but has not yet been announced. We expect these rows to be filled in quickly once this initial PR is merged, so I think `-` makes more sense.

Reply:
Ah, good thinking. Makes sense to me. 👍